openmicroscopy/openmicroscopy
# Omero reader update : Open OMERO ROIs in ImageJ

To Test:

1) Open Insight : Add roi's to an existing OMERO image and save (All kinds of ROI's).
2) Now close Insight, and download/setup the Insight-IJ plugin, open the same OMERO image on ImageJ using this plugin.
3) Please check the "Display ROIs","Display OME-XML metadata" and "Display Metadata" checkbox's in the Bio-Formats Import options and press OK.
4) This should now open the IMAGE from OMERO in ImageJ. And the ROI's should be loaded onto the ImageJ ROI Manager.


NOTE: The label text ROI's will be displayed in the metadata, but the support for displaying the same in the ImageJ ROIManager will be done via a separate PR.
